internal/backends/nodejs/nodejs_test.go

```diff
@@ -0,0 +1,112 @@
+package nodejs
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/replit/upm/internal/api"
+)
+
+func TestNodejsYarnBackend_Guess(t *testing.T) {
+	tcs := []struct {
+		scenario    string
+		backend     api.LanguageBackend
+		fileContent string
+		expected    map[api.PkgName]bool
+	}{
+		{
+			scenario: "Returns the imports when given a JavaScript file with normal imports",
+			backend:  NodejsNPMBackend,
+			fileContent: `
+			import React from 'react';
+
+			export const App = () => null;
+		`,
+			expected: map[api.PkgName]bool{
+				"react": true,
+			},
+		},
+		{
+			scenario: "Ignore internal imports",
+			backend:  NodejsNPMBackend,
+			fileContent: `
+			const http = require('http');
+		`,
+			expected: map[api.PkgName]bool{},
+		},
+		{
+			scenario: "Ignore local file imports",
+			backend:  NodejsNPMBackend,
+			fileContent: `
+			import React from 'react';
+			import SomeComponent from './SomeComponent';
+
+			export const App = () => React.createElement(SomeComponent, {});
+		`,
+			expected: map[api.PkgName]bool{
+				"react": true,
+			},
+		},
+		{
+			scenario: "Ignore https file imports",
+			backend:  NodejsNPMBackend,
+			fileContent: `
+			import React from "https://cdn.skypack.dev/react";
+			import SomeComponent from './SomeComponent';
+
+			export const App = () => React.createElement(SomeComponent, {});
+		`,
+			expected: map[api.PkgName]bool{},
+		},
+		{
+			scenario: "Will process packages in namespace",
+			backend:  NodejsNPMBackend,
+			fileContent: `
+			import React from 'react';
+			import { Button } from '@material-ui/core';
+			
+			export const App = () => React.createElement(Button, { color: "primary" }, "Hello, World!");
+		`,
+			expected: map[api.PkgName]bool{
+				"react":             true,
+				"@material-ui/core": true,
+			},
+		},
+	}
+
+	for _, tc := range tcs {
+		t.Run(tc.scenario, func(t *testing.T) {
```

Comment:
```suggestion
	for _, tc := range tcs {
		tc := tc
		t.Run(tc.scenario, func(t *testing.T) {
```

This probably doesn't matter too much since the subtest aren't marked as parallel. But in case we do mark it as parallel at some point it might be nice to do this to save us from https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721